package.json

```diff
@@ -51,7 +51,7 @@
     "underscore": "^1.8.3"
   },
   "devDependencies": {
-    "babel": "^5.8.21",
+    "babel": "^5.8",
```

Comment:
Also, it's actually preferred to include the patch version, since it indicates what the *minimum* patch version we require is.